runtimes/server-interface/chat.ts

```diff
@@ -35,8 +33,6 @@ export type Chat = {
     onTabChange: (handler: NotificationHandler<TabChangeParams>) => void
     onTabRemove: (handler: NotificationHandler<TabRemoveParams>) => void
     onVote: (handler: NotificationHandler<VoteParams>) => void
```

Comment:
Currently, it's based on telemetry. Do you think it can change in the future and voting becomes server concern?

Reply:
onVote can also be removed